openwisp_users/tests/test_admin.py

```diff
@@ -169,6 +172,36 @@ def test_admin_change_user_email_empty(self):
         self.assertEqual(len(mail.outbox), 0)
         self.assertContains(response, 'errors field-email')
 
+    def test_admin_change_user_page_get_invalid_UUID(self):
+        admin = self._create_admin()
+        self.client.force_login(admin)
+        with self.subTest('Test for wrong identifier'):
+            response = self.client.get(
+                reverse(f'admin:{self.app_label}_user_change', args=['WRONG']),
+                follow=True,
+            )
+            # TODO: Remove this when we drop support for django<3.0
+            if django.VERSION < (3, 1):
+                # check for version < 3.1
+                content = 'User with ID "WRONG" doesn\'t exist. Perhaps it was deleted?'
+            else:
+                content = 'User with ID “WRONG” doesn’t exist. Perhaps it was deleted?'
+            response.content = html.unescape(response.content.decode())
+            self.assertContains(response, content, status_code=200)
+        with self.subTest('Test for non-existing user'):
+            id = uuid.uuid4()
+            response = self.client.get(
+                reverse(f'admin:{self.app_label}_user_change', args=[id],), follow=True
+            )
+            # TODO: Remove this when we drop support for django<3.0
+            if django.VERSION < (3, 1):
+                # check for version < 3.1
+                content = f'User with ID "{id}" doesn\'t exist. Perhaps it was deleted?'
+            else:
+                content = f'User with ID “{id}” doesn’t exist. Perhaps it was deleted?'
+            response.content = html.unescape(response.content.decode())
```

Comment:
You don't need to do this here. You can remove it.

Reply:
Are you referring to line no 202?

Reply:
Yes!

Reply:
It is required for Django 2.2, since it converts HTML entities like `&quot;` to `"`

Reply:
Thanks @Saurav-Shrivastav, but support for django 2.2 was already planned to be dropped, the PR was ready but not merged yet, now is merged: https://github.com/openwisp/openwisp-users/pull/199.